project/src/test/java/com/google/sps/SearchStoreTest.java

```diff
@@ -38,66 +41,157 @@
   private static final String DESCRIPTION_WITH_GAMES =
       "Sutter Middle School will be walking to McKinley Park. 7th grade class and teachers will"
           + " have a picnic, play games, and eat lunch at the park and Clunie Pool.";
-  private static final ArrayList<Keyword> KEYWORDS_NAME_WITHOUT_GAMES =
-      new ArrayList<Keyword>(Arrays.asList(new Keyword("picnic", 1.00f)));
-  private static final ArrayList<Keyword> KEYWORDS_NAME_WITH_GAMES =
-      new ArrayList<Keyword>(
-          Arrays.asList(new Keyword("picnic", 0.56f), new Keyword(GAMES, 0.44f)));
-  private static final ArrayList<Keyword> KEYWORDS_DESCRIPTION_WITHOUT_GAMES =
-      new ArrayList<Keyword>(
-          Arrays.asList(
-              new Keyword("Sutter Middle School", 0.43f),
-              new Keyword("McKinley Park", 0.14f),
-              new Keyword("teachers", 0.10f),
-              new Keyword("class", 0.10f),
-              new Keyword("picnic", 0.09f),
-              new Keyword("park", 0.08f),
-              new Keyword("lunch", 0.03f),
-              new Keyword("Clunie Pool", 0.03f)));
-  private static final ArrayList<Keyword> KEYWORDS_DESCRIPTION_WITH_GAMES =
-      new ArrayList<Keyword>(
-          Arrays.asList(
-              new Keyword("Sutter Middle School", 0.41f),
-              new Keyword("McKinley Park", 0.13f),
-              new Keyword("teachers", 0.09f),
-              new Keyword("class", 0.09f),
-              new Keyword("picnic", 0.09f),
-              new Keyword("park", 0.08f),
-              new Keyword("lunch", 0.07f),
-              new Keyword(GAMES, 0.01f),
-              new Keyword("Clunie Pool", 0.03f)));
+   private static final ImmutableList<Keyword> KEYWORDS_NAME_WITHOUT_GAMES =
+      ImmutableList.of(new Keyword("picnic", 1.00f));
+  private static final ImmutableList<Keyword> KEYWORDS_NAME_WITH_GAMES =
+      ImmutableList.of(new Keyword("picnic", 0.56f), new Keyword(GAMES, 0.44f));
+  private static final ImmutableList<Keyword> KEYWORDS_DESCRIPTION_WITHOUT_GAMES =
+      ImmutableList.of(
+          new Keyword("Sutter Middle School", 0.43f),
```

Comment:
Make these keywords into constants since they're used quite a few times?

Reply:
Yes, some keywords are similar - I think here since the constants are the actual NLP output, it's easier to see the output without constants.